validator/main.py

```diff
@@ -95,7 +102,7 @@ def get_toxicity(self, value: str) -> List[str]:
         # with confidence higher than the threshold
         pred_labels = []
         if value:
-            results = self._model.predict(value)
```

Comment:
I think this won't work because this same code is running on the endpoint. See the app.py, this loop here only needs to run when calling inference_local